war/src/main/js/handlebars-helpers/id.js

```diff
@@ -0,0 +1,3 @@
+export default function id(str) {
+	return (''+str).replace(/\W+/g, '_');
```

Comment:
Does it matter that potentially multiple values share the same ID with this replacement scheme?

Reply:
TBH I don't know. I limited the scope of this PR to just maintain current behaviour. I extracted the function as it is.

Reply:
I'm resolving this one, feel free to reopen it if you disagree!